package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ushahidi-platform-pattern-library",
   "description": "Ushahidi Platform Pattern Library",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
@middle8media This should have been `3.5.1-pre.1` or something similar. It shouldn't get a real version until we're absolutely ready to go to production. Once a version is pushed to npm you can't overwrite it.. meaning if any more changes are needed before this goes to production we have to skip 3.5.1 straight to 3.5.2. 

Also looks like this was pushed to npm, but not tagged in git?